src/statd/python/yanger/yanger.py

```diff
@@ -498,7 +515,7 @@ def add_container(containers):
         # The 'podman ps' command lists ports even in host mode, but
         # that's not applicable, so skip networks and port forwardings
         networks = container_inspect(container["name"], ("NetworkSettings", "Networks"))
```

Comment:
If `container_inspect()` was modified to always just return the whole object (thereby also removing the need for `getitem()`), or an empty `dict` on error, then I think this line could be:

```python
networks = container_inspect(container["name"]).get("NetworkSettings", {}).get("Networks", {})
```

Reply:
Nice one, I'll look into it!